project/src/main/world/environment/MarshEnvironment.tscn

```diff
@@ -33,7 +33,6 @@
 [ext_resource path="res://src/main/world/ObstacleSpawner.tscn" type="PackedScene" id=42]
 [ext_resource path="res://src/main/world/environment/ButtercupStoolSpawnerW.tscn" type="PackedScene" id=43]
 [ext_resource path="res://assets/main/world/environment/buttercup-stool-green.png" type="Texture" id=45]
-[ext_resource path="res://src/main/world/obstacle-map.gd" type="Script" id=46]
```

Comment:
Describe why obstacle-map wasn't replaced with an invisible-obstacle-autotiler in the commit comment.

Reply:
Expanded commit comment, `...these are the only environments used in free roam mode....`